packages/icons/src/upstream-artifact.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="none" xmlns:v="https://vecta.io/nano"><path d="M14 11.855v-.64-.64c0-.192-.126-.32-.316-.32H9.958v.32c0 .544-.411.961-.947.961H6.989c-.537 0-.947-.416-.947-.961v-.32H2.316c-.189 0-.316.128-.316.32v3.106c0 .16.126.32.316.32h.632-.293c-.088-.099-.088-.099-.338 0h11.368c.175 0 .197-.099.316-.32v-.979c0-.352-.074-.37 0-.847zm-1.074-.064c.189 0 .316.128.316.32 0 .16-.126.32-.316.32s-.316-.16-.316-.32.126-.32.316-.32zm-1.263 0c.19 0 .316.128.316.32 0 .16-.126.32-.316.32s-.316-.16-.316-.32.126-.32.316-.32zm-3.439 1.544l-4.645-.567V14h10.105c.189 0 .316-.16.316-.32v-1.825l-.684.913-5.092.567zm-1.55-5.002h2.653v1.281H6.674V8.332zm0 1.921h2.653v.32a.32.32 0 0 1-.316.32H6.989c-.189 0-.316-.128-.316-.32v-.32zm4.484-3.843H9.326v1.281H6.674V6.411H4.842c-.284 0-.411-.32-.253-.544L7.747 2.12c.126-.16.347-.16.474 0l3.158 3.746c.19.224.063.544-.221.544z" fill="#000"/></svg>
```

Comment:
can we use currentColor instead of hardcoded color?

Reply:
updated